recipes/tttrlib/build.sh

```
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+mkdir b2 && cd b2
+
+if [[ "${target_platform}" == osx-* ]]; then
```

Shellcheck

[warning] target_platform is referenced but not assigned.

(SC2154)